### DISCUSSION TITLE
what is core going to  do after cache miss or data dependecy ?

### DISCUSSION THREAD

**Discussion by syncmanner** [jbush001/NyuziProcessor#210] [General]
According to the doc description , core will issue next thread after current thread .  Is arbiter  be blocked by current thread ?  Is pipeline like instruction cache tag  blocked by current thread ?  what is the interface of adiacent pipeline, it has any conceptions like outstanding ?

**Comment by jbush001** [comment]
Nothing blocks. The arbiter chooses a new thread from the ready ones every cycle. If there is no thread available, it will basically issue a "bubble" or empty instruction. Each thread is in-order issue.
